pkg/middleware/auth_proxy.go

```diff
@@ -96,14 +96,14 @@ func initContextWithAuthProxy(store *remotecache.RemoteCache, ctx *models.ReqCon
 	ctx.IsSignedIn = true
 
 	// Remember user data in cache
-	if err := auth.Remember(id); err != nil {
+	if e := auth.Remember(id); e != nil {
```

Comment:
err is renamed to e since it's not of the error type, which is expected of variables named err.